galaxy-integration-steam.pyproj

```diff
@@ -0,0 +1,189 @@
+    <Compile Include="tests\tests_steam_network\conftest.py" />
+    <Compile Include="tests\tests_steam_network\test_achievements.py" />
+    <Compile Include="tests\tests_steam_network\test_authentication.py" />
+    <Compile Include="tests\tests_steam_network\test_data.py" />
+    <Compile Include="tests\tests_steam_network\test_friends.py" />
+    <Compile Include="tests\tests_steam_network\test_friends_cache.py" />
+    <Compile Include="tests\tests_steam_network\test_game_time.py" />
+    <Compile Include="tests\tests_steam_network\test_owned_games.py" />
+    <Compile Include="tests\tests_steam_network\test_owned_games_cache.py" />
+    <Compile Include="tests\tests_steam_network\test_presence.py" />
+    <Compile Include="tests\tests_steam_network\test_protobuf_client.py" />
+    <Compile Include="tests\tests_steam_network\test_protocol_client.py" />
+    <Compile Include="tests\tests_steam_network\test_user_info_cache.py" />
+    <Compile Include="tests\tests_steam_network\test_websocket_client.py" />
+    <Compile Include="tests\tests_steam_network\test_websocket_list.py" />
+    <Compile Include="tests\tests_steam_network\__init__.py" />
+    <Compile Include="tests\test_backend_switch.py" />
+    <Compile Include="tests\test_cache.py" />
+    <Compile Include="tests\test_calling_unsupported_method.py" />
+    <Compile Include="tests\test_custom_config.py" />
+    <Compile Include="tests\test_features.py" />
+    <Compile Include="tests\test_integration.py" />
+    <Compile Include="tests\test_local_games.py" />
+    <Compile Include="tests\test_local_size.py" />
+    <Compile Include="tests\test_plugin_version_cache.py" />
+    <Compile Include="tests\test_user_profile_checker.py" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="protobuf_files\" />
+    <Folder Include="protobuf_files\proto\" />
+    <Folder Include="requirements\" />
+    <Folder Include="src" />
+    <Folder Include="src\commonWeb\" />
+    <Folder Include="src\commonWeb\css" />
+    <Folder Include="src\commonWeb\fonts\" />
+    <Folder Include="src\commonWeb\fonts\LatoWeb" />
+    <Folder Include="src\commonWeb\img" />
+    <Folder Include="src\steam_network" />
+    <Folder Include="src\steam_network\custom_login" />
+    <Folder Include="src\steam_network\custom_login\css" />
+    <Folder Include="src\steam_network\protocol" />
+    <Folder Include="src\steam_network\protocol\messages\" />
+    <Folder Include="tests" />
+    <Folder Include="tests\tests_public_profiles" />
+    <Folder Include="tests\tests_steam_network" />
+  </ItemGroup>
+  <ItemGroup>
+    <Interpreter Include="env1\">
+      <Id>env1</Id>
+      <Version>3.7</Version>
+      <Description>env1 (Python 3.7 (64-bit))</Description>
+      <InterpreterPath>Scripts\python.exe</InterpreterPath>
+      <WindowsInterpreterPath>Scripts\pythonw.exe</WindowsInterpreterPath>
+      <PathEnvironmentVariable>PYTHONPATH</PathEnvironmentVariable>
+      <Architecture>X64</Architecture>
```

Comment:
It should be 32 bit, yes. That's a visual studio file, it's not used to actually run the code, so the settings there don't really matter. I originally kept it out of the project but every time i changed branches I'd lose it and need to recreate it, which was annoying. So it's in the repo

Reply:
yeah if its in `.gitignore` it shouldn't dissapear when changing branches

Reply:
When changing branches it'll remove all files not in the new branch, and that includes the vs files. I guess i could have wrapped it all in a parent directory and put the vs files there so it didn't happen,  but I'm inclined to leave them in because it's convenient for other VS/VS Code users.

Reply:
oh interesting that alr then